bedrock/base/templates/macros-protocol.html

```diff
@@ -280,22 +257,15 @@
 {% macro content_card(name, content_cards=None) -%}
   {% set card_data = get_content_card(content_cards or page_content_cards, name) -%}
   {% if card_data -%}
+    {% set image = '<img class="mzp-c-card-image" src="' ~ card_data.image_url ~ '" srcset="' ~ card_data.highres_image_url ~ ' 1.5x">' %}
```

Comment:
Issue (blocking): I think we might need to make this a little more robust still as `card_data.highres_image_url` is optional.

http://localhost:8000/fr/ throws a 500 error `KeyError 'highres_image_url'`

Reply:
Added an `{% if card_data.highres_image_url %}` check below